CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### New features
+
+* [#10100](https://github.com/rubocop/rubocop/issues/10100): Add new `Style/NumberedParameters` cop. ([@Hugo-Hache][])
+
```

Comment:
You already created the `changelog/` file, so this is unneeded!

Reply:
Indeed, fixed